packages/labs/analyzer/src/lib/lit/template.ts

```diff
@@ -0,0 +1,584 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview
+ *
+ * Utilities for analyzing lit-html templates.
+ */
+
+import {
+  isCommentNode,
+  isDocument,
+  isDocumentFragment,
+  isElementNode,
+  traverse,
+} from '@parse5/tools';
+import {_$LH} from 'lit-html/private-ssr-support.js';
+import {parseFragment, type DefaultTreeAdapterTypes, type Token} from 'parse5';
+import type ts from 'typescript';
+import {
+  isLitHtmlImportDeclaration,
+  isResolvedPropertyAccessExpressionLitHtmlNamespace,
+} from './modules.js';
+
+export {
+  isCommentNode,
+  isDocumentFragment,
+  isElementNode,
+  isTextNode,
+} from '@parse5/tools';
+
+const {getTemplateHtml, marker, markerMatch, boundAttributeSuffix} = _$LH;
+
+type TypeScript = typeof ts;
+
+// Why, oh why are parse5 types so weird? We re-export them to make them easier
+// to use.
```

Comment:
you should be able to pull these from `@parse5/tools` itself now

i think it re-exports the defaults